docs/section5/Rule5-1.md

```diff
@@ -0,0 +1,64 @@
+# Envelope - Rule 5-1  
+**Schema Version** 0.0.33
+**Primary Rule** True
+**Rule ID:** 5-1  
+**Rule Description:** There are four baseline rotations (i.e., four baseline models differing in azimuth by 90 degrees and four sets of baseline model results) if vertical fenestration area per each orientation differ by more than 5%.
+**Rule Assertion:** Options are PASS/FAIL     
+**Appendix G Section:** Table G3.1#5a baseline column   
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each RMD 
+
+**Applicability Checks:**  
+None
+   
+**Function Calls:**  
+1. get_opaque_surface_type()  
+
+
+## Rule Logic:   
+
+Get the fenestration area for each unique orientation (i.e., azimuth) and then check if the minimum and maximum areas differ by 5% or more
+- Create a blank dictionary that will have all unique azimuths paired with the total vertical fenestration area: `azimuth_fen_area_dict = {}`  
+- For each building in the B_RMI: `for bldg in B_RMI.buildings:`    
+    - For each building_segment in the bldg: `for bldg_seg in bldg.building_segments`      
+        - For each zone in the building_segment: `for zone in bldg_seg.zones`   
+            - For each surface in zone: `for surface in zone.surfaces:`  
+                - Check if surface is above-grade wall: `if get_opaque_surface_type(surface) == "ABOVE-GRADE WALL":`   
+                    - Get the azimuth: `surface_azimuth = surface.azimuth`  
+                    - Check if the azimuth is not in a key in the dictionary of azimuths and fen area, if it is not then add it: `if surface_azimuth not in azimuth_fen_area_dict:`   
+                        - Add the unique azimuth to the dictionary as a key and set initial fen area to 0: `azimuth_fen_area_dict[surface_azimuth]= 0`  
```

Comment:
Shall we come up with a map that maps a range of azimuth degrees to an orientation instead of using azimuth as the key.

Reply:
My understanding was 5% variation based on four rotation orientation (0. 90. 180. 270)
Azimuth could be any number between 0- 360. Using azimuth as a dictionary key may result in more than 4 orientations.

Reply:
Thanks Weili. By the definition in 90.1 orientation is not confined to 0, 90, 180, 270 so I think it is okay as is. Let's let the others weigh in on it and see what their opinions area.

Reply:
I agree with Christina based on this 90.1 definition: 
orientation: the direction an envelope element faces, i.e., the direction of a vector perpendicular to and pointing away from the surface outside of the element. 

So the exception #2 to not rotate baseline for "...Buildings where the vertical fenestration area on each orientation varies by less than 5%." in my opinion requires determining the fenestration area for each unique azimuth and then confirming that this area varies by no more that 5%. E.g., finding the azimuths with the max and the min fenestration area, and confirming that the two areas differ by no more than 5% in order to not have to rotate.